extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -15,6 +15,7 @@
  */
 
 import {IntersectionObserver} from '../../../src/intersection-observer';
+import {isPositionFixed} from '../../amp-ad/0.1/amp-ad-3p-impl';
```

Comment:
Ping @dvoytenko. I think he wrote similar code before.

On Fri, Jul 22, 2016 at 1:52 PM, Yuxi Chen <notifications@github.com> wrote:

> In extensions/amp-iframe/0.1/amp-iframe.js
> <https://github.com/ampproject/amphtml/pull/4163#discussion_r71944299>:
>
> > @@ -15,6 +15,7 @@
> >   */
> >
> >  import {IntersectionObserver} from '../../../src/intersection-observer';
> > +import {isPositionFixed} from '../../amp-ad/0.1/amp-ad-3p-impl';
>
> @cramforce <https://github.com/cramforce> Which file should I factor into?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ampproject/amphtml/pull/4163/files/7057b4bc0016b1e8e39762564305b308788bcab1#r71944299>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAFeTxfwrJmrDgg03JshvT90LcVLLTwZks5qYS2VgaJpZM4JSRbM>
> .
>